pom.xml

```diff
@@ -604,7 +604,7 @@ THE SOFTWARE.
         <configuration>
           <signature>
             <groupId>org.codehaus.mojo.signature</groupId>
-            <artifactId>java1${java.level}</artifactId>
+            <artifactId>java16</artifactId>
```

Comment:
The justification being that just as we do not want people using Java 7+ APIs in the near term, we do not want them writing new code using `-source 7` language constructs just yet.